docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/configuring.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Configuring Grafana Mimir deployment with Jsonnet"
+menuTitle: "Configuring deployment"
+description: "Learn how to configure Grafana Mimir when using Jsonnet."
+weight: 20
+---
+
+# Configuring Grafana Mimir deployment with Jsonnet
+
+Notable features of the Mimir Jsonnet are described here in detail.
+To learn how to get started, see [Deploying Grafana Mimir with Jsonnet and Tanka]({{< relref "./deploying.md" >}}).
+
+## Anti-affinity
+
+Given the distributed nature of Mimir, both performance and reliability are improved when pods are spread across different nodes.
+For example, multiple queriers can be processing the same query at the same time, so it's better to distribute them across different nodes. Morever, since losing multiple ingesters can cause data loss, it's also important to have them spread.
```

Comment:
```suggestion
For example, multiple queriers can simultaneously process the same query, so it's better to distribute them across different nodes. Moreover, because losing multiple ingesters can cause data loss, it's also important to spread the queriers out.
```
In the last sentence, "them" is ambiguous as is: does it refer to queriers or ingesters?